org.eclipse.scanning.api/src/org/eclipse/scanning/api/event/scan/NexusSlitsPosition.java

```diff
@@ -0,0 +1,86 @@
+/*-
+ * Copyright © 2017 Diamond Light Source Ltd.
+ *
+ * This file is part of GDA.
+ *
+ * GDA is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License version 3 as published by the Free
+ * Software Foundation.
+ *
+ * GDA is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with GDA. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.eclipse.scanning.api.event.scan;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * This is a special Position class to allow both x and y gap values to be returned from a
+ * NexusSlitsWrapper scannable. It is a temporary creation until we understand why can't
+ * use a List or a Map for a DeviceValue in a DeviceRequest. 
+ *
+ */
+public class NexusSlitsPosition {
+	private double x_gap;
+	private double y_gap;
+
+	public double getX_gap() {
+		return x_gap;
+	}
+
+	public void setX_gap(double x_gap) {
+		this.x_gap = x_gap;
+	}
+
+	public double getY_gap() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)